dotnet/aspnetcore/kubernetes/Application/aspnet-core-dotnet-core/charts/sampleapp/templates/service.yaml

```diff
@@ -8,7 +8,11 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 spec:
+  {{- if .Values.ingress.enabled }}
+  type: ClusterIP
+  {{- else }}
   type: {{ .Values.service.type }}
```

Comment:
Should this be hard-coded as well. Otherwise we do not seem to be honoring the values file above which is confusing